pymc_marketing/mmm/base.py

```diff
@@ -30,36 +29,36 @@
 __all__ = ("BaseMMM", "MMM")
 
 
-class BaseMMM:
+class BaseMMM(ModelBuilder):
     model: pm.Model
 
     def __init__(
         self,
-        data: pd.DataFrame,
         target_column: str,
         date_column: str,
         channel_columns: Union[List[str], Tuple[str]],
+        data: Optional[pd.DataFrame] = None,
         channel_prior: Optional[TensorVariable] = None,
         validate_data: bool = True,
+        model_config: Optional[Dict] = None,
+        sampler_config: Optional[Dict] = None,
```

Comment:
It would be nice to add docstrings to this class since it is unclear what these arguments mean if one is unfamiliar with the `ModelBuilder` class.